docs/FAQ.md

```diff
@@ -137,6 +137,20 @@ commit, then run `jj restore --from Y --to @-` to restore the parent commit
 to the old state, and `jj restore --from X` to restore the new working-copy
 commit to the new state.
 
+### How do I resume working on an existing change?
+
+There are two ways to resume working on an earlier change: `jj new` then `jj squash`,
+and `jj edit`. The first is the recommended default, but `jj edit` can be useful.
```

Comment:
nit: Maybe "generally recommended" instead of "the recommended default" so the reader doesn't think they should configure it as their default somehow

Reply:
Sure, will use